src/core/annotation.js

```diff
@@ -704,6 +708,47 @@ class Annotation {
     }
   }
 
+  setRotation(mk) {
+    this.rotation = 0;
+    if (mk instanceof Dict) {
+      let angle = mk.get("R") || 0;
+      if (Number.isInteger(angle) && angle !== 0) {
+        // If the angle is negative then angle % 360 is still negative.
+        angle %= 360;
+        angle = angle >= 0 ? angle : 360 + angle;
```

Comment:
Can't we just do *exactly* what's done elsewhere in the code-base, since re-assigning the value when it's non-negative just feels unnecessary?
```suggestion
        // Normalize the rotation, by clamping it to the [0, 360) range.
        angle %= 360;
        if (angle < 0) {
          angle += 360;
        }
```